app/services/find_nearby.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class FindNearby
+  AVAILABLE_PLACES = %w[car_dealer car_rental car_wash car_repair gas_station parking].freeze
+  def find_places(latitude, longitude)
+    places = {}
```

Comment:
Use 'tap'. Looks much cleaner. Alicja learned me this :)
```
{}.tap do |places|
    AVAILABLE_PLACES.each do |place|
      places[place] = GoogleApi.find_objects_nearby(latitude, longitude, place)['results']
    end
end
```

Reply:
Good tip Fabian! :)